docs/specs/assets/2026-08-11-employee-history-mockup.html

```
@@ -0,0 +1,417 @@
+<title>Employee History — People App</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)